files/ceph-version-qemu.sh

```diff
@@ -59,7 +59,7 @@ for YUM_HISTORY_EVENT in $YUM_HISTORY_EVENTS; do
 done
 
 # Retrieve a list of all QEMU process IDs
-PIDS=$(pidof /usr/libexec/qemu-kvm)
+PIDS=$(pgrep -f /usr/libexec/qemu-kvm)
```

Comment:
I would change the command to `pgrep -f "^/usr/libexec/qemu-kvm"`

Reason: with the original command, pgrep will look for all commands that contain "/usr/libexec/qemu-kvm". This might include other processes that are not of your interest, e.g., if the same command is run from ansible, it will also take into account the spawned process from ansible. By adding `^` you can focus only on the processes whose command starts with the path to that binary, i.e., the processes that we are actually interested in.

Reply:
Noted with thanks. I will check and push a new commit.